mypy.ini

```diff
@@ -10,7 +10,7 @@ disallow_any_generics = True
 disallow_incomplete_defs = True
 disallow_subclassing_any = True
 ; disallow_untyped_calls = True
-disallow_untyped_decorators = True
+disallow_untyped_decorators = False
```

Comment:
Why this ?

Reply:
To avoid the `pytest.mark` warning. It's the only decorator I have.